opengever/workspace/workspace_meeting.py

```diff
@@ -88,6 +88,15 @@ class IWorkspaceMeetingSchema(model.Schema):
         default=list()
     )
 
+    form.order_after(guests='attendees')
+    guests = schema.List(
+        title=_(u"label_guests", default=u"Guests"),
+        required=False,
+        value_type=schema.ASCIILine(),
+        default=[],
```

Comment:
beware of mutable defaults:
```suggestion
        default=list(),
        missing_value=list(),
```

Reply:
Resolved as suggested